### DISCUSSION TITLE
Current instance [ac190b7088616-cdh3-ea-com1] not listed as an open instance: []

### DISCUSSION THREAD

**Discussion by roamerop** [JanusGraph/janusgraph#2486] [Q&A] [answered]
I want to get openInstances, and close it , but i falied to get the result.  
JanusGraph verson:0.5.3, Hbase backend + Es index
```
gremlin> :> graph.openManagement().isOpen()
==>true
gremlin> :> graph.openManagement().getOpenInstances()
Current instance [ac190b7088616-cdh3-ea-com1] not listed as an open instance: []
Type ':help' or ':h' for help.
Display stack trace? [yN]    
```

**Comment by li-boxuan** [comment, marked as answer]
Can you provide all commands you did from opening the graph, to this exception? Can you please also provide your configuration file? Thanks

**Reply by roamerop** [reply]
I solved the problem.  do not use `:>` send command.
gremlin>  graph = JanusGraphFactory.build.set("storage.backend","hbase").set(....).open()
gremlin>  graph.openManagement().getOpenInstances()
